web/packages/shared/components/UnifiedResources/shared/PinButton.tsx

```diff
@@ -79,11 +96,12 @@ export function PinButton({
 
 function getTipContent(
   pinningSupport: PinningSupport,
-  pinned: boolean
+  pinned: boolean,
+  notSupportedTipContent?: string
 ): string {
   switch (pinningSupport) {
     case PinningSupport.NotSupported:
-      return PINNING_NOT_SUPPORTED_MESSAGE;
+      return notSupportedTipContent || PINNING_NOT_SUPPORTED_MESSAGE;
```

Comment:
I think you can get rid of `notSupportedTipContent` and just add the new "17.whatever or newer" message instead. Pinning resources has been supported since 14.1 so we dont need to expect that error message to ever show again.